include/vcpkg/base/messages.h

```diff
@@ -1373,6 +1465,14 @@ namespace vcpkg
         "{value} may be either a 'vendor' like 'Azure' or 'NuGet', or a file path like C:\\example or /usr/example",
         "Restored {count} package(s) from {value} in {elapsed}. Use --debug to see more details.");
     DECLARE_MESSAGE(ResultsHeader, (), "Displayed before a list of installation results.", "RESULTS");
+    DECLARE_MESSAGE(SecondLinkerMemberTooSmallArchiveMem,
+                    (),
+                    "",
+                    "Second linker member was too small to contain the expected number of archive members");
+    DECLARE_MESSAGE(SecondLinkerMemberTooSmallUint32,
+                    (),
+                    "",
+                    "Second linker member was too small to contain a single uint32_t");
```

Comment:
Maybe `uint32_t` needs explanation as well.

Reply:
This looks more like development `[DEBUG]` messages than messages we should show to users.